src/Controls/src/Core/SetterSpecificityList.cs

```diff
@@ -25,15 +25,19 @@ public void SetValue(SetterSpecificity specificity, object value)
 			{
 				_first = new KeyValuePair<SetterSpecificity, object>(specificity, value);
 				if (_values is not null)
```

Comment:
that would require locking on something else than _values

Reply:
Like an `object _valueLocker = new()` as a field and lock that?

Reply:
Maybe that is needed because what happens if the code path hits this at the same time and the `_values` is set twice? There is no lock around the assignment of the field.